lineage/export_states.py

```diff
@@ -0,0 +1,146 @@
+""" This file exports excel sheets of lineages of cells with their states assigned to them after fitting. """
```

Comment:
How much do the results change here from fitting run to fitting run?

Reply:
Honestly, I don't know, but I think it is a lot. I set a random seed in figure 11, too, so it wouldn't change, but haven't set it in the file that runs the fitting and creates this export.